src/components/FilterBar/FilterDisplay/Inputs/DateInput.react.js

```diff
@@ -0,0 +1,73 @@
+export class DateInput extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { value: this.props.filterBarActor.getFilter(this.props.filterUid).value || { from: null, to: null } };
+  }
+
+  _onChange(event) {
+    var newValue = this.state.value;
+
+    if(event.type === 'dp') {
```

Comment:
A really annoying necessity from the bootstrap-datetime-picker we're using.

Changing the date by using the datepicker generates a different event than just using the input field and entering text. The datetime-picker generates dp events that don't have a target or value, so I can't just use event.target.value.